src/app/api/run/route.ts

```diff
@@ -0,0 +1,19 @@
+import { NextResponse } from 'next/server'
+import fs from 'fs'
+import path from 'path'
+import { UPLOAD_DIR } from '@/app/utils'
+
+export async function GET() {
+    try {
+        // Get all files in the upload directory
+        const files = await fs.promises.readdir(UPLOAD_DIR)
+
+        // Filter only .csv files
```

Comment:
Why only filter on CSV files? It seems like the setup app only cares if runs have successfully completed

Reply:
i thought the assumption for the pilot would be we're only dealing with .csv files but we can keep this simpler and just return all files - some of the code is making assumptions when they parse the CSV file 🤔 

ie in `saveFile` we save it as `.csv`, maybe thats too narrow of a scope? or ok for now?

Reply:
If we're assuming that it'll only be CSV files then we don't need to add code that checks if it's a CSV file